### PR TITLE
Wpl serialization

### PR DESCRIPTION
**Pull request by TermanEmil**
Serialization is made using `System.Xml.Serialization`.

`WplParts.*` know how to construct themselves from `WplPlaylist`. For example `RawPlaylist` has a copy constructor which receives a `WplPlaylist`.

Fixed a platform specific bug with WPL and Wpl directories.

### CODE REVIEW

File: src/NPlaylist/Wpl/WplParts/Head.cs [hunk 1]
```diff
@@ -16,5 +16,27 @@ public class Head
...
+            Author = wplPlaylist.Author;
+            Title = wplPlaylist.Title;
+        }
+
+        private static List<Meta> ExtractHeadMeta(WplPlaylist playlist)
+        {
+            return playlist
+                .Tags
+                .Where(kv =>
+                    kv.Key != TagNames.Author &&
```
Comment (aivascu) [inline src/NPlaylist/Wpl/WplParts/Head.cs]: move the logical operators to the left side

**Comment by aivascu** [comment]
@TermanEmil have you solved the issue with the directory structure?

**Comment by TermanEmil** [comment]
Yes

**Comment by aivascu** [comment]
@TermanEmil please update the base branch

**Pull request merged**